tests/models/mamba2/test_modeling_mamba2.py

```diff
@@ -92,6 +101,11 @@ def __init__(
         self.num_choices = num_choices
         self.scope = scope
         self.tie_word_embeddings = tie_word_embeddings
+        self.classifier_dropout = classifier_dropout
+
+    # TODO: This might fail, need to create an internal testing
+    def get_large_model_config(self):
+        return Mamba2Config.from_pretrained("state-spaces/mamba2-2.7b")
```

Comment:
This won't work, we need to convert everything to hf format first and I don't see why we need it (at least now).

Reply:
Makes sense.